includes/widgets/social-icons.php

```diff
@@ -607,7 +607,7 @@ protected function render() {
 		$migration_allowed = Icons_Manager::is_migration_allowed();
 
 		?>
-		<div class="elementor-social-icons-wrapper elementor-grid">
+		<div class="elementor-social-icons-wrapper elementor-grid" role="list">
```

Comment:
@rami-elementor same question like in the share-buttons, what if the user add only one button

Reply:
@arielk Fixed.